docs/CURL-DISABLE.md

```diff
@@ -12,6 +12,26 @@ Disable support for HTTP cookies.
 
 Disable support for authentication methods using crypto.
 
+## `CURL_DISABLE_BASIC_AUTH`
+
+Disable support for the Basic authentication methods.
+
+## `CURL_DISABLE_BEARER_AUTH`
+
+Disable support for the Bearer authentication methods.
+
+## `CURL_DISABLE_DIGEST_AUTH`
+
+Disable support for the Digest authentication methods.
+
+## `CURL_DISABLE_NEGOTIATE_AUTH`
+
+Disable support for the negotiate authentication methods.
+
+## `CURL_DISABLE_AWS`
+
+Disable AWS-SIG4 support.
```

Comment:
```suggestion
Disable **AWS-SIG4** support.
```

I think this fixes the spelling complaint.

Reply:
Yep, that fixed it.